fury/tests/test_interactor.py

```diff
@@ -141,11 +141,14 @@ def test_double_click_events(recording=False):
     filename = "test_double_click_events.log.gz"
     recording_filename = pjoin(DATA_DIR, filename)
 
-    cube1 = actor.cube([(0, 0, 0)], [(0.16526678, 0.0186237, 0.01906076)],
+    label = vtk.vtkOpenGLBillboardTextActor3D()
```

Comment:
By curiosity Why you did not pick one of our actors (label actor or text3D actor)?

Reply:
No reason, I found it as an example in the repo when looking for `set_color` because I couldn't make the cube change color! (I'm guessing it's because it uses a shader, maybe?)

Reply:
I changed it for a `ui.TextBlock2D` instead. ^_^

Reply:
👍 cool! thanks 😄